state/accountBlockInfo.go

```diff
@@ -0,0 +1,27 @@
+package state
+
+type accountBlockInfo struct {
+	nonce    uint64
+	hash     []byte
+	rootHash []byte
+}
+
+// GetNonce returns the block nonce
```

Comment:
optional: the block's ...

Reply:
also, why using getters when this struct could have remained a DTO

Reply:
Removed, since it was renamed to `common.blockInfo`.